### ISSUE TITLE
Icon: update confirmation-icon to latest g4 color

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#1416]
The current green (#36CF57) is failing a11y contrast. Let's update all confirmations icon to the new g4 colour of #28A443.

![Screen Shot 2021-04-26 at 1 43 59 PM](https://user-images.githubusercontent.com/38065/116148228-7ba9de00-a695-11eb-839b-e7ecb47b4ac7.png)

**Comment by ianmcburnie** [comment]
Done in 12.4.0
